slackbridge/utils.py

```diff
@@ -182,7 +129,7 @@ def emoji_replace(match: Match[str]) -> str:
     text = re.sub(r'<\@(U\w+)\|?(\w+)?>', user_replace, text)
     text = re.sub(r'<!(\w+)\|?(\w+)?>', var_replace, text)
     text = re.sub(r'<(?!!)([^|]+?)>', lambda match: match.group(1), text)
-    text = re.sub(r':([\d\w+-]+):', emoji_replace, text)
+    text = re.sub(r':([\d\w+-]+):', emojize(match.group(1)), text)
```

Comment:
You'll need this to be `lambda match: emojize(match.group(1))` instead, otherwise it won't have the `match` variable available to use.